Doc/howto/functional.rst

```diff
@@ -57,7 +47,7 @@ functional, for example.
 
 In a functional program, input flows through a set of functions. Each function
 operates on its input and produces some output.  Functional style discourages
-functions with side effects that modify internal state or make other changes
+functions with *side effects* that modify internal state or make other changes
```

Comment:
This document mostly uses strong-emphasis to indicate a term being defined. To make it consistent:

```suggestion
functions with **side effects** that modify internal state or make other changes
```